packages/@webex/plugin-meetings/src/rtcMetrics/index.ts

```diff
@@ -1,5 +1,26 @@
+import anonymizeIP from 'ip-anonymize';
 import RTC_METRICS from './constants';
 
+/**
+ * Anonymize identifiable information. For now, only IP addresses.
+ *
+ * @param {array} stats - An RTCStatsReport organized into an array of strings.
+ * @returns {void}
+ */
+const removePii = (stats: string[]): string[] => {
+  return stats.map((statLine) => {
+    const data = JSON.parse(statLine);
+    // on local and remote candidates, anonymize the last 4 bits.
+    if (data.type === 'local-candidate' || data.type === 'remote-candidate') {
```

Comment:
Can we wrap this in a function called "anonymizeIP" which takes data as a parameter and returns a new data object (stringified) with all the checks internally. It's easier to test sections and contain the statements.

Reply:
Right, added.